src/AirshipBindings.iOS.Core/StructsAndEnums.cs

```diff
@@ -49,7 +49,9 @@ public enum UALogLevel : long
 		Warn = 2,
 		Info = 3,
 		Debug = 4,
-		Trace = 5
+		[Obsolete("Use Verbose instead. Trace will be removed in a future release.")]
+		Trace = 5,
+		Verbose = 5
```

Comment:
Duplicate enum values are allowed in C#, so I did this in order to avoid a breaking change. Maybe slightly funky, but we can get rid of it in the next major.